src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
```

Comment:
@jmophoto That is a good point. We should keep the `type` to `text` for now. It definitely wouldn't work right with `type` being `checkbox`, `file`, `radio`, etc.